frontend/src/app/components/ExperimentCardList.js

```diff
@@ -27,3 +31,16 @@ export default function ExperimentCardList({
     </div>
   );
 }
+
+function setCurrentExperiments(isDev, allExperiments) {
+  const currentExperiments = [];
+  const utcNow = moment.utc();
+  if (isDev) return allExperiments;
+  allExperiments.forEach((experiment) => {
+    if (moment(utcNow).isAfter(experiment.launch_date) || typeof experiment.launch_date === 'undefined') {
+      currentExperiments.push(experiment);
+    }
+  });
+  return currentExperiments;
+}
```

Comment:
I won't bust your chops over putting this here, but its a good candidate to be a selector on the experiments reducer.